src/sentry/migrations/0664_monitor_drop_last_state_change_db.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 5.0.2 on 2024-03-06 18:06
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0663_artifactbundleindex_cleanup_step3"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                ALTER TABLE "sentry_monitorenvironment" DROP COLUMN "last_state_change";
+                """,
+                    reverse_sql="""
+                ALTER TABLE "sentry_monitorenvironment" ADD COLUMN "last_state_change" timestamptz;
```

Comment:
Add it as nullable here. Mostly for dev but this reverse will fail otherwise

Reply:
I think default it is nullable without `NOT NULL` right?

Reply:
Refrence: https://www.postgresql.org/docs/7.3/sql-altertable.html#:~:text=The%20new%20column%20always%20comes,to%20set%20the%20default%20afterwards.

> In the current implementation of ADD COLUMN, default and NOT NULL clauses for the new column are not supported. The new column always comes into being with all values NULL. You can use the SET DEFAULT form of ALTER TABLE to set the default afterwards.

right?

Reply:
I always get this backwards, because in Django it's the opposite!